scripts/fetch-numpydeppkg.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Script to fetch and install python package that depends on numpy
+# This script has been tested for eht-imaging, gwpy and pycbc packages
+# For more details on eht-imaging refer to the links below:
+# https://github.com/achael/eht-imaging
+# https://achael.github.io/_pages/imaging/
+# For more details on gwpy see: https://github.com/gwpy/gwpy
+# For more details on pycbc see: https://github.com/gwastro/pycbc
+
+# The following function does the following:
+# 1. Fetches latest software sources for the specified package in the given  working dir
+#    (default current-dir/numpy_<pkgname>_dep)
+# 2. Assumes python3, virtualenv, python and pip3 are available and installs graphviz, pipdeptree
+# 3. Git clones specified package
+#
+# To generate the dependency graph for the specified package and highlight the
+# role of NumPy use the script gen-numpy-dep-graph.sh
+# 
+
```

Comment:
It would be good to use `set -euxo pipefail` here, so the scripts fails at the point something goes wrong. It took me a while to figure out I was missing `virtualenv` ...

Reply:
ok, will do.

Reply:
done.